src/chart/generateCategoricalChart.tsx

```diff
@@ -1103,6 +1103,70 @@ export const generateCategoricalChart = ({
         mouseHandlerCallback: this.triggeredAfterMouseMove,
         layout: this.props.layout,
       });
+      this.displayDefaultTooltip();
+    }
+
+    displayDefaultTooltip() {
+      const { children, data, height, layout } = this.props;
+
+      const tooltipElem = findChildByType(children, Tooltip);
+      // If the chart doesn't include a <Tooltip /> element, there's no tooltip to display
+      if (!tooltipElem) {
+        return;
+      }
+
+      const { defaultIndex } = tooltipElem.props;
+
+      // Protect against runtime errors
+      if (typeof defaultIndex !== 'number' || defaultIndex < 0 || defaultIndex > this.state.tooltipTicks.length) {
+        return;
+      }
+
+      const activeLabel = this.state.tooltipTicks[defaultIndex] && this.state.tooltipTicks[defaultIndex].value;
+      let activePayload = getTooltipContent(this.state, data, defaultIndex, activeLabel);
+
+      const independentAxisCoord = this.state.tooltipTicks[defaultIndex].coordinate;
+      const dependentAxisCoord = (this.state.offset.top + height) / 2;
+
+      const isHorizontal = layout === 'horizontal';
+      let activeCoordinate = isHorizontal
+        ? {
+            x: independentAxisCoord,
+            y: dependentAxisCoord,
+          }
+        : {
+            y: independentAxisCoord,
+            x: dependentAxisCoord,
+          };
+
+      // Unlike other chart types, scatter plot's tooltip positions rely on both X and Y coordinates. Only the scatter plot
+      // element knows its own Y coordinates.
+      // If there's a scatter plot, we'll want to grab that element for an interrogation.
+      const scatterPlotElement = this.state.formattedGraphicalItems.find(
```

Comment:
I wish we didn't have to do special handling for this, but understand why. Just letting it be known